ddl/migrations/0188_wheel_spin_configs.sql

```diff
@@ -0,0 +1,22 @@
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS wheel_spin_configs (
```

Comment:
I think this table isn't super necessary. The prizes themselves could refer directly to the mint and I think we'd save ourselves some work / complexity

Reply:
yeah i don't think we need to have it but i see your point. i'm okay doing it but maybe instead of wheels we could call it a prize group or something